.github/workflows/python-build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build Python Package
+
+on: [pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
```

Comment:
You might want to add `fail-fast: false` here so if one of the builds fails the others will continue.